hack/build-bytecode-images-multi.sh

```diff
@@ -25,7 +25,7 @@ PROGRAMS='{
 MAPS='{
 "direct_flows":"ringbuf",
 "aggregated_flows":"hash",
-"additional_flow_metrics":"per_cpu_hash"
+"additional_flow_metrics":"per_cpu_hash",
```

Comment:
Assuming this is JSON, could we pass the value through jq to validate the content? This would ensure:
- The issue cannot silently reoccur
- Any errors are surfaced earlier, perhaps with a self-test in the script itself

Untested: 

```bash
echo "$MAPS" | jq empty || { echo "Invalid JSON in MAPS"; exit 1; }
```

Reply:
this metadata added to image so not sure where u will run the above ?

Reply:
this is good idea will add

Reply:
Ditto for `PROGRAMS`.

Reply:
Should have been: 

```bash
echo "$MAPS" | jq --exit-status || { echo "Invalid JSON in MAPS"; exit 1; }
```